kimchi/src/snarky/checked_runner.rs

```diff
@@ -404,4 +410,7 @@ where
             panic!("woot");
         }
     }
+    pub fn poseidon(&mut self, preimage: (CVar<F>, CVar<F>)) -> (CVar<F>, CVar<F>) {
+        super::poseidon::poseidon(self, preimage)
+    }
```

Comment:
looks good, but please add a space between functions, and documentation as well :o

Reply:
I'm guessing `cargo fmt` is going to yell at you anyway?

Reply:
I thought that too, but it seems that `cargo fmt` accepts either 0 or 1 spaces. I'll add it.